src/ImmersedBoundaries/ImmersedBoundaries.jl

```diff
@@ -100,17 +100,19 @@ abstract type AbstractImmersedBoundary end
 ##### ImmersedBoundaryGrid
 #####
 
-struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
+struct ImmersedBoundaryGrid{FT, TX, TY, TZ, G, I, M, Arch} <: AbstractGrid{FT, TX, TY, TZ, Arch}
     architecture :: Arch
     underlying_grid :: G
     immersed_boundary :: I
+    wet_cells_map :: M
```

Comment:
What's the different between an "active cell" and a "wet cell"? What is the type of the "map"?

Reply:
The map is a `arch_array` of tuples of unsigned integers, where
```
const MAXUInt8  = 2^8  - 1
const MAXUInt16 = 2^16 - 1
const MAXUInt32 = 2^32 - 1

N = maximum(size(grid))
Type = N > MAXUInt8 ? (N > MAXUInt16 ? (N > MAXUInt32 ? UInt64 : UInt32) : UInt16) : UInt8
```

`wet_cell` is the opposite of `immersed_cell`.  We could rename it to `inactive_cell`. Then we have to figure out what this means for partial cells, but I guess it won't be too difficult

Reply:
I have renamed all instances of "wet" to "active"